src/wwwroot/app/services/VacancyService.js

```diff
@@ -45,6 +45,13 @@ export default class VacancyService {
       _UserService = UserService;
    }
 
+   getVacancy(id) {
+      let additionalUrl = VACANCY_URL + id;
+      return _HttpService.get(additionalUrl).then((vacancy) => {
```

Comment:
you can do something like this
`return _HttpService.get(additionalUrl).then(this.convertFromServerFormat)`